Gems/Prefab/PrefabDependencyViewer/Code/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# If we are on a host platform, we want to add the host tools targets like the PrefabDependencyViewer.Editor target which
+# will also depend on PrefabDependencyViewer.Static
+if(PAL_TRAIT_BUILD_HOST_TOOLS)
+    ly_add_target(
+        NAME PrefabDependencyViewer.Editor.Static STATIC
+        NAMESPACE Gem
+        AUTOMOC
+        FILES_CMAKE
+            prefabdependencyviewer_editor_files.cmake
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                Source
+            PUBLIC
+                Include
+        BUILD_DEPENDENCIES
+            PUBLIC
+                AZ::AzToolsFramework
+            PRIVATE
+                Legacy::Editor.Headers
+                Gem::GraphCanvasWidgets
+    )
+
+    ly_add_target(
+        NAME PrefabDependencyViewer.Editor GEM_MODULE
+        NAMESPACE Gem
+        AUTOMOC
+        OUTPUT_NAME Gem.PrefabDependencyViewer.Editor
+        FILES_CMAKE
+            prefabdependencyviewer_editor_shared_files.cmake
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                Source
+            PUBLIC
+                Include
+        BUILD_DEPENDENCIES
+            PUBLIC
+                Gem::PrefabDependencyViewer.Editor.Static
+    )
+
+    # By default, we will specify that the above target PrefabDependencyViewer would be used by
+    # Tool and Builder type targets when this gem is enabled.  If you don't want it
+    # active in Tools or Builders by default, delete one of both of the following lines:
+    ly_create_alias(NAME Prefab.Tools    NAMESPACE Gem TARGETS Gem::PrefabDependencyViewer.Editor)
+    ly_create_alias(NAME Prefab.Builders NAMESPACE Gem)
```

Comment:
It is not necessary to specify the "empty" Builders alias.  This means that you can probably delete the Prefab.Builders line and adjust the comment above to indicate that its only there for Tools

Reply:
The "empty" Builders alias is used to add the Gem Source Path to the generated `cmake_dependencies.<project>.assetprocessor.setreg` file containing, which is also used to the "<gem-root>/Assets" directory as a scan folder for the AssetProcessor.

This is there because the Prefab gem contains that single graphcanvas_style.json asset.